grammar.js

```diff
@@ -949,6 +950,19 @@ module.exports = grammar(C, {
       repeat1(choice($.raw_string_literal, $.string_literal))
     ),
 
+	ud_suffix: $ => token.immediate(/[a-zA-Z_]\w*/),
```

Comment:
```suggestion
	udl_suffix: $ => token.immediate(/[a-zA-Z_]\w*/),
```
missed an "l" :)

Reply:
Hmm, "ud-suffix" seems to be the official name for this in the ISO standard. See: https://github.com/cplusplus/draft/blob/c+%2B17/source/lex.tex#L1732-L1761

Reply:
This is pretty minor, but I usually prefer to use unabbreviated words in the rule names (`declaration` instead of `decl`, etc).

What do you think about renaming this to `literal_suffix`? I know it doesn't communicate the "user-defined" part of the name, which `ud` stands for, but that will already be a part of the parent node (`user_defined_literal`).

@cynix @jdrouhard thoughts?

Reply:
I also prefer the spelled out names, so I like `literal_suffix` more. `ud_suffix` does match the standard though, so there's a case for both. To be consistent with the rest of this grammar, spelling it out is the way to go so that's my vote.

Reply:
Makes sense, there's certainly value in consistency. Updated.